test/system/255-auto-update.bats

```diff
@@ -0,0 +1,243 @@
+# This test can fail in dev. environment because of SELinux.
+# quick fix: chcon -t container_runtime_exec_t ./bin/podman
+@test "podman auto-update - label io.containers.autoupdate=image" {
+    run_podman images
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top image
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman auto-update
+    is "$output" "Trying to pull.*" "Image is updated."
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    [[ "$output" != "$ori_image" ]]
+}
+
+@test "podman auto-update - label io.containers.autoupdate=disabled" {
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top disabled
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman auto-update
+    is "$output" "" "Image is not updated with disabled."
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    is "$output" "$ori_image" "Image hash should not changed."
+}
+
+@test "podman auto-update - label io.containers.autoupdate=fakevalue" {
+    fakevalue=$(random_string)
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top $fakevalue
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman ? auto-update
+    is "$output" ".*invalid auto-update policy.*" "invalid policy setup"
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    is "$output" "$ori_image" "Image hash should not changed."
+}
+
+@test "podman auto-update with multiple services" {
+    fakevalue=$(random_string)
+    run_podman inspect --format "{{.Id}}" $IMAGE
+    img_id="$output"
+    cnames=()
+    cnames_update=()
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top image
+    cname_update="${lines[0]}"
+    cnames+=($cname_update)
+    cnames_update+=($cname_update)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine_nginx:latest top image
+    cname_update_1="${lines[0]}"
+    cnames+=($cname_update_1)
+    cnames_update+=($cname_update_1)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top
+    cname_default="${lines[0]}"
+    cnames+=($cname_default)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top disabled
+    cname_disabled="${lines[0]}"
+    cnames+=($cname_disabled)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top ""
+    cname_empty="${lines[0]}"
+    cnames+=($cname_empty)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top $fakevalue
+    cname_fakevalue="${lines[0]}"
+    cnames+=($cname_fakevalue)
+    echo $output
```

Comment:
All these `echo`s are not useful.

Also, you could accomplish the same thing with much less duplication by:
```bash
   for label in image "" disabled $fakevalue; do
      cnames+=$(generate_service alpine $label)
   done
```

Reply:
OK. Will make a loop for this.

Reply:
And the line of echo $output will help tester get the output from the functions. So we will get useful information from the test log to analyses the test failed in which step.